packages/@okta/i18n/src/properties/login_da.properties

```diff
@@ -930,6 +940,14 @@ oie.symantecVip.challenge.title = Bekr\u00E6ft med {0}
 oie.symantecVip.challenge.description = Indtast den genererede sikkerhedskode fra {0}-appen
 oie.symantecVip.verify.passcode.label = Angiv sikkerhedskode
 
+## YubiKey Authenticator
+oie.yubikey.label = YubiKey Authenticator
```

Comment:
Shouldn't "Authenticator" need to be translated here?

Reply:
I have the same question. It seems to be translated in certain languages (nb, sv). Not sure if this is a language specific thing.

Reply:
The English loan word "authenticator" entered the technical term lexicon in these languages. This is expected.